### DISCUSSION TITLE
Documentation Configuration & Issue

### DISCUSSION THREAD

**Discussion by ddkasa** [Textualize/textual#5413] [Q&A]
I have been meaning to fix the broken formatting at https://textual.textualize.io/api/containers/#textual.containers.ItemGrid and https://textual.textualize.io/api/containers/#textual.containers.VerticalScroll, but I can't seem to figure out which config ends up in the final deployment. 

I have tried the suggestion from the contributing docs & the online config, but they seem to give a different result, as the parent params are not getting shown on child objects?

Also is there a way to prevent mkdocs from re-screenshotting everything again on every change or is that inevitable?

**Comment by willmcgugan** [comment]
I run `make docs-serve`. That will cache screenshots after the first run. Still takes 30 seconds to rebuild the docs though.

**Reply by ddkasa** [reply]
Thanks, this still doesn't solve the issue of the child params though, as I can't see them with this command?

# Local
![2024-12-19 19 43 20 127 0 0 1 def38f3021b3](https://github.com/user-attachments/assets/81c38e0c-e38e-46f8-94ab-2725959a7a69)

# Deployed 
![2024-12-19 19 44 38 textual textualize io e06ddb69acff](https://github.com/user-attachments/assets/6f22a6f8-8ca1-403b-ba8d-9299adb10efa)
